1.7/administration/installing/cloud/gce.md

```diff
@@ -54,7 +54,7 @@ You must create the RSA public/private keypairs to allow passwordless logins via
 1.  Open RSA pub key:
 
     ```bash
-    $ vi ~/.ssh/id_rsa.pub
+    $ sudo vi ~/.ssh/id_rsa.pub
```

Comment:
`~/.ssh/id_rsa.pub` should be owned by the current user, so sudo not necessary here.